lib/push.js

```diff
@@ -177,6 +181,39 @@ module.exports = function (log, db) {
     },
 
     /**
+     * Notifies every device in the account except the current one that the
+     * password was changed
```

Comment:
IIRC it's possible for Firefox to receive webchannel messages from content-server even if it's not the "current device".  I can have a content-server page open in Firefox, sending webchannel messages to Firefox, but using a different sessionToken than the one that Firefox itself is using for sync.  In this case the browser would receive both the webchannel message and the push notification.

> the action we do on the browser when we get a reset pwd push
> message is to reset the cache credentials and transition the browser UI in "please reconnect" mode

How much messier would it be to have the browser check its sessionToken, and only enter "please reconnect" if the sessionToken has actually been invalidated?  To me this seems a little more robust than counting on three separate components (auth-server, content-server and Firefox) to maintain a consistent opinion on who gets notified of what when.